### PR TITLE
Hw sp 02

### PR DESCRIPTION
**Pull request by DmGrt**

### CODE REVIEW

**Review by fyarkiy — APPROVED** [review]
Good job!

File: src/main/java/com/springhw/config/AppConfig.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.springhw.service",
+        "com.springhw.dao"
+})
+public class AppConfig {
+    @Autowired
+    private Environment env;
```
Comment (RomanDubovskyi) [inline src/main/java/com/springhw/config/AppConfig.java:23]: let's user constructor injection

Comment (RomanDubovskyi) [inline src/main/java/com/springhw/config/AppConfig.java:23]:
```suggestion
    private Environment environment;
```

**Review by RomanDubovskyi — COMMENTED** [review]
change PR name it's uninformative 
and remove files from previous hw, they shouln't be in this PR

**Pull request closed without merging**